src/components/ChatLog.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+const ChatLog = (props) => {
+	const entries = props.entries;
+
+	const chatComponents = entries.map((message, index) => {
+		return (
+			<div className="chat-log" key={message.id}>
+				<ChatEntry
+					id={message.id}
+					sender={message.sender}
+					body={message.body}
+					timeStamp={message.timeStamp}
+					liked={message.liked}
+					onUpdate={props.onUpdateHeart}
+				></ChatEntry>
+			</div>
+		);
+	});
+	return chatComponents;
```

Comment:
This works! HTML-wise, it is creating a _lot_ of extra `div` elements that we could clean up by doing the following:
```python
const chatComponents = entries.map((message, index) => {
    return (
        <ChatEntry
            key={message.id}
            id={message.id}
            sender={message.sender}
            body={message.body}
            timeStamp={message.timeStamp}
            liked={message.liked}
            onUpdate={props.onUpdateHeart}
        />
    );
});
return (
    <div className="chat-log">
        {chatComponents}
    </div>
);
```
Now there is only one `div` parent surrounding all the `ChatEntry` components